packages/ignition/src/app/error.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+import Logo from "@/components/logo";
+import OnError from "./on-error";
+
+const Error = () => (
```

Comment:
Maybe there is something funny going on with the error page? 

This is how it rendered for me in Opera/Chrome.

<img width="2001" alt="image" src="https://github.com/rocketclimb/rocketicons/assets/1039567/06581d7d-c658-4f10-8f0a-57a4e1d45e3a">

Reply:
No, it's look right. We must working on that, but having just a white page as we have now it's terrible for the users, so I've putted this simple one on place